release-notes/major9/09.19.0/valtimo-frontend-libraries.md

```diff
@@ -20,6 +20,10 @@ The following features were added:
 
     The default `v-select` component's background style color appearance has been changed to white to add contrast and distinguish it from the disabled state.
 
+* **Added auto scrolls modal to the top**
+
+  The `valtimo-modal` component now has automatic scrolling to the top of the modal when transitioning between pages.
```

Comment:
I would change this to: 'The valtimo-modal component now scrolls to the top when transitioning between form flow steps, and when switching pages in a [Form.IO](http://form.io/) wizard'